localization/english/events/event_localization/aeluran_story_events_l_english.yml

```diff
@@ -1,17 +1,17 @@
 ﻿l_english:
-  aeluran_story.001.title:1 "Crimson Visitors"
-  aeluran_story.001.desc:0 "Theres a commotion in your holding. Sisters of the Aeluran Order, known as the red witches by the common folk, have showed up on your doorstep and are demanding your council. They say the know of your discovery on your last expedition."
-  aeluran_story.001.ok:0 "Ill hear them in my council chambers, steward see them in" 
+    aeluran_story.001.title:1 "Crimson Visitors"
+    aeluran_story.001.desc:0 "As you begin to settle down for the night, a servant bursts into your chambers in a rush. Outside, you hear a grand commotion as servants and guards alike rush to welcome your unexpected new guests. Sisters of the Aeluran Order, known as the red witches by the common folk, have somehow learnt of your latest expedition and are demanding your council."
```

Comment:
I like the overall change but Im hung up on "a servant bursts into your chambers in a rush", maybe "a servant knocks at your door with great urgency"?